harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -282,6 +279,15 @@ def _run(self) -> None:
                         ),
                         "stop_requested": self.context.get_stop_requested(),
                     }  # type: workload.Response
+                    if (
+                        self.context.distributed.size > 1
+                        and not self.context._average_training_metrics
+                    ):
+                        warnings.warn(
+                            "Only the chief worker's training metrics are being reported, due "
+                            "to setting average_training_metrics to False.",
+                            FutureWarning,
```

Comment:
this shouldn't be a future warning.  Maybe make it a `UserWarning` or a `RuntimeWarning`.  Docs [here](https://docs.python.org/3/library/warnings.html#warning-categories)